waspc/data/Generator/templates/react-app/scripts/validate-env.mjs

```diff
@@ -1,4 +1,4 @@
-import { throwIfNotValidAbsoluteURL } from './universal/validators.mjs';
+import { throwIfNotValidAbsoluteURL } from 'wasp/universal/validators';
```

Comment:
Since this file is now a part of an NPM package (the SDK):
- We no longer need to copy this file into both the web app and the server.
- We also no longer need to perform the `mjs` stuff.